spec/uploaders/alert_image_uploader_spec.rb

```diff
@@ -3,81 +3,4 @@
 
 RSpec.describe AlertImageUploader do
   include CarrierWave::Test::Matchers
```

Comment:
Would it make sense to delete this file? Or do you think you'll add to it in the future?

Reply:
Removed in d74292d4 -- I'll reintroduce if needed